TinyConsole/TinyConsole.swift

```diff
@@ -26,22 +26,52 @@ open class TinyConsole {
         return self.dateFormatter.string(from: Date())
     }
     
-    public static func scrollToBottom(){
+    public static func scrollToBottom() {
         if let textView = shared.textView, textView.bounds.height < textView.contentSize.height {
             textView.layoutManager.ensureLayout(for: textView.textContainer)
             let offset = CGPoint(x: 0, y: (textView.contentSize.height - textView.frame.size.height))
             textView.setContentOffset(offset, animated: true)
         }
     }
     
-    public static func print(_ text: String, global: Bool = true){
+    public static func print(_ text: String, global: Bool = true, color : UIColor = UIColor.white) {
+        
+        let formattedText = NSMutableAttributedString(string: text)
+        let range = NSRange(location: 0, length: formattedText.length)
+        
+        // set text font and color
+        if let font = UIFont(name: "Menlo", size: 12.0){
```

Comment:
Just a small performance thing, don't you think it would be better if the font was only created once and not every time the print function is called? 

I'm just thinking about a scenario where you have a relative big project and you print everything to TinyConsole. You have stuff running in the background etc. You might be calling the print function multiple times a second if it's doing an intensive debug task and recreating the same font object each time.

The same would be true for line 58 in this PR.

Maybe I'm just "over optimising" here, I'll leave you guys decide if you want to implement this suggestion. :)

Other than that, thanks for implementing coloured logs! :D